src/pseudo-backend/buildings/Buildings.constants.ts

```diff
@@ -0,0 +1,27 @@
+import { BuildingsMapping, PlanetTypes } from './Buildings.types';
+
+export const buildingsMapping: BuildingsMapping = {
```

Comment:
@DylanBozarth @linuxuser07 This is where we will define the requirements for each building. I've made the typings pretty strict here on initial development, but we can loosen them up later if we want to.  We can add whatever we want here including min/max resources from the system, and system types that we want to include or exclude. (Right now its something like "Radioactive" and "Gaia" but we could add other things like "Under Revolution", "Under Coup" for example if we want to do some checks that way.